source/extensions/filters/http/jwt_authn/jwks_async_fetcher.cc

```diff
@@ -22,7 +27,34 @@ JwksAsyncFetcher::JwksAsyncFetcher(const RemoteJwks& remote_jwks,
                                    JwtAuthnFilterStats& stats, JwksDoneFetched done_fn)
     : remote_jwks_(remote_jwks), context_(context), create_fetcher_fn_(create_fetcher_fn),
       stats_(stats), done_fn_(done_fn), cache_duration_(getCacheDuration(remote_jwks)),
-      debug_name_(absl::StrCat("Jwks async fetching url=", remote_jwks_.http_uri().uri())) {
+      debug_name_(absl::StrCat("Jwks async fetching url=", remote_jwks_.http_uri().uri())),
+      num_retries_(RetryCount) {
+
+  uint64_t base_interval_ms = RetryInitialDelayMilliseconds;
+  uint64_t max_interval_ms = RetryMaxDelayMilliseconds;
+
+  if (remote_jwks_.has_retry_policy()) {
```

Comment:
Hmm,  I prefer to implememt the retry in [here](https://github.com/envoyproxy/envoy/blob/main/source/extensions/filters/http/common/jwks_fetcher.cc#L33)

both on-demand fetch and background fetch will benefit.

Reply:
just pushed a an additional commit that implements it in jwks_fetcher.cc